bool camera_left
bool camera_right
bool camera_up
bool camera_down

int16 Servo_shake
int16 Servo_node

bool take_picture
